utils/sound.py

```diff
@@ -0,0 +1,23 @@
+from playsound import playsound
+
+# mp3 files created from: https://ttsmp3.com/
+
```

Comment:
We should probably just zip the .mp3 files and throw them on discord instead of in the project.